src/bin/compute-gop.cc

```diff
@@ -130,16 +133,17 @@ int main(int argc, char *argv[]) {
 
     po.Read(argc, argv);
 
-    if (po.NumArgs() != 4 && po.NumArgs() != 5) {
+    if (po.NumArgs() != 6) {
```

Comment:
This line is problematic.

The help message at line 111 says it takes 5 required positional arguments and 1 optional one. So 5 is also a valid value. But this line requires that the number of arguments has to be exactly 6.

I am wondering how you have tested the changes.

Reply:
Thank you for reviewing my code. I got it, thank you for your explanation, I fix this condition latter.

Reply:
I just realized it is a problem with the original code.
I think the `[]` in `[<phone-feature-wspecifier>]` can be removed. The code does actually need it as a required argument.